perun-web-gui/src/main/java/cz/metacentrum/perun/webgui/json/usersManager/ChangeNonAuthzPassword.java

```diff
@@ -77,6 +78,38 @@ public void onLoadingStart() {
 
 	}
 
+	/**
+	 * Changes password for the user
+	 *
+	 * @param token
+	 * @param password new password to set
+	 */
+	public void changeNonAuthzPassword(String token, String password) {
```

Comment:
This whole class is not used by admin GUI, you can probably remove it completely.

Reply:
Removed